### PR TITLE
Update CHANGELOG and VERSION for 2.5.0 release

### PR DESCRIPTION
**Pull request by JLXIA**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Release 2.5.0 19-02-2020
+
+- Auto-generate JWT Audience if no audiences are set in service config
+- Retry after failed token fetch
+- handle disable_auth in oneplatform service config
```
Comment (nareddyt) [inline CHANGELOG.md]: Please capitalize for consistency

Comment (nareddyt) [reply CHANGELOG.md]: `disable_auth` with backticks

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+- Add listener_port and http2_port flags
```
Comment (nareddyt) [inline CHANGELOG.md]: `listener_port`, `http2_port`

File: CHANGELOG.md (hunk 1 above)
Comment (qiwzhang) [inline CHANGELOG.md]: don't call oneplatform.  call it endpoint service config

File: CHANGELOG.md (hunk 2 above)
Comment (JLXIA) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (JLXIA) [reply CHANGELOG.md]: Done

Comment (JLXIA) [reply CHANGELOG.md]: Done

**Pull request merged**